images/linux/scripts/installers/preparemetadata.sh

```diff
@@ -8,6 +8,10 @@
 source $HELPER_SCRIPTS/document.sh
 
 WriteItem "<!--- DO NOT EDIT - This markdown file is autogenerated. -->"
+
+Announcements=$(cat $HELPER_SCRIPTS/UbuntuAnnouncements.md)
```

Comment:
I think it is better to name file just `announcements.md` because it is located in `linux` folder and it is obvious that it is for ubuntu.
The same for windows

Reply:
Done